src/controller/controller.py

```diff
@@ -73,7 +74,24 @@ def create_ant(self, identifier):
         def _create_ant():
             time.sleep(all_params.controller_params.create_ant_time)
             nest = self.game_state.get_nests()[0]
-            self.game_state.create_ants(nest, amount=1)
+            self.game_state.create_ants(nest, amount=1, ant_type='worker')
+            self.view.increment_ant_count(type=button.ant_type)
+
+        create_thread(func=_create_ant)
+
+    def create_scout(self, identifier):
+        """
+        Event-handler for creating scouts using the create scouts button
+        :param identifier: identifier for create scout button
+        :return: nothing
+        """
+        button = self.view.get_element_by_id(identifier)
+        button.state = 'loading'
+
+        def _create_ant():
```

Comment:
def_create_ant() is duplicated (will be used in create_scout, create_worker, create_soldier, create_superant), so it might be worth taking out to a _private_method.

Reply:
What I mean with this is a method called _create_ant(ant_type) in the class, not in each method.